src/main/java/ch/njol/skript/structures/StructCommand.java

```diff
@@ -315,10 +318,9 @@ public boolean postLoad() {
 
 	@Override
 	public void unload() {
-		if (scriptCommand != null) {
-			Commands.unregisterCommand(scriptCommand);
-			syncCommands.set(true);
-		}
+		assert scriptCommand != null; // This method should never be called if one of the loading methods fail
```

Comment:
Again, this is different from the previous behaviour since in a production environment it will just continue with the value `null`.

Reply:
I would prefer an error instead of silently doing nothing as this means the methods have been abused or not called properly.